### DISCUSSION TITLE
record > retain > all

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15032] [Ask A Question] [answered]
Looking at the Full Refrence config `record.retain.all` is an optional config. I cant seems to figure out what the defult is if nothing is configured. I am assuming `all` is the default but just checking.

Also is there is a way we can help contribute to update the config page to have missiong information like this?

**Comment by hawkeye217** [comment, marked as answer]
Many of the recording retain options have changed for 0.15, currently in beta. So I don't think we plan to make any updates to the existing docs at this point.

**Reply by MrCaspan** [reply]
but for any documents missing information will we just post on here to let you guys know and you guys would choose to add it or is there a way to contribute if we can?

**Reply by hawkeye217** [reply]
You're always welcome to ask in a discussion, but you can also [create a pull request](https://github.com/blakeblackshear/frigate/pulls) if you feel something needs updating and we can discuss more there.
